src/runtime/internal/lifecycle.ts

```diff
@@ -11,18 +11,47 @@ export function get_current_component() {
 	return current_component;
 }
 
+/**
+ * Schedules a callback to run immediately before the component is updated after any state change.
+ * 
+ * The first time the callback runs will be before the initial `onMount`
+ * 
+ * https://svelte.dev/docs#run-time-svelte-beforeupdate
```

Comment:
i'm thinking maybe use the `{@see}` jsdoc for links?

```suggestion
 * @see {@link https://svelte.dev/docs#run-time-svelte-beforeupdate}
```

Reply:
<img width="689" alt="image" src="https://user-images.githubusercontent.com/35429197/189597951-1575feef-e62c-448a-9587-d2f74ce2be09.png">

<img width="697" alt="image" src="https://user-images.githubusercontent.com/35429197/189598166-61ea8bd3-5ed5-4b98-91ed-3a82d6c902b2.png">

Makes close to no difference, at least in VSCode. I can add the JSDoc annotation if desired. The only downside I see is an additional (minor) syntax burden for contributors. What is JSDoc, what tags exist, etc. instead of writing plain markdown.

Reply:
yea i know, it's more of a nitpick to suggest to follow the jsdoc convention.

Reply:
You are one of the maintainers. I can add JSDoc if the PR will be merged.